stories/Popover/Popover.story.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  AutoExample,
+  Tab,
+  Example,
+  Title,
+  StoryContainer,
+} from '@catho/quantum-storybook-ui';
+
+import Popover from '../../components/Popover';
+import { Container } from '../../components/Grid';
+import PopoverExampleTop from './examples/PopoverExampleTop';
+import PopoverExampleLeft from './examples/PopoverExampleLeft';
+import PopoverExampleBottom from './examples/PopoverExampleBottom';
+import PopoverExampleRight from './examples/PopoverExampleRight';
+
+const TabExample = (
+  <Tab title="Examples">
```

Comment:
Could you do the "Examples" tab story with a layout like [Input](https://github.com/catho/quantum/blob/master/stories/Input/Input.story.jsx#L13)?

💅

Reply:
Done.